CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ ExternalProject_Add(taglib
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wno-unused-result -Wno-deprecated-declarations")
 
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -no-pie")
```

Comment:
Why the addition of `-no-pie`?

Reply:
I wasn't able to run the program without the -no-pie flag on Arch Linux, not sure what the root cause was.

Reply:
Weird -- I'll have to look into this, but I do my Linux development primarily on Manjaro, which is an Arch-variant, and haven't had any problems. PIE is a good security feature, so we probably don't want to disable it. Did you have problems compiling before the addition of your plugin?

Reply:
Yup, it wouldn't run at all unless I enabled that flag, although I could build it. There might be another solution to this, I didn't look too far into the issue.